prog/test/hetzner_server.rb

```diff
@@ -37,21 +37,12 @@ def self.assemble(vm_host_id: nil)
   label def fetch_hostname
     update_stack({"hostname" => hetzner_api.get_main_ip4})
 
-    hop_add_ssh_key
-  end
-
-  label def add_ssh_key
-    keypair = SshKey.generate
-    hetzner_api.add_key("ubicloud_ci_key_#{strand.ubid}", keypair.public_key)
-    update_stack({"hetzner_ssh_keypair" => Base64.encode64(keypair.keypair)})
-
     hop_reimage
   end
 
   label def reimage
     hetzner_api.reimage(
       frame["server_id"],
-      hetzner_ssh_key: hetzner_ssh_keypair.public_key,
```

Comment:
We still need to pass public key (of the temporary global ssh key), no?

Reply:
`reimage` takes a default of `Config.hetzner_ssh_public_key` for that argument, so its passed